src/data_tools/data_transformer.py

```diff
@@ -0,0 +1,867 @@
+
+    # center
+    radius = 5
+    random.seed(seed + 100)
+    a = random.randint(radius + 50, HEIGHT - radius - 51) 
+    random.seed(seed + 200)
+    b = random.randint(radius + 50, WIDTH - radius - 51)
+    cv2.circle(ret,(b,a), radius, (255,255,255), -1)
+
+    pts1 = np.array([[b,a],[0,0],[25, 0]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[140,0],[170, 0]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[400,0],[420, 0]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[1120,0],[1160, 0]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[WIDTH - 1,0],[WIDTH - 1, 25]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[WIDTH - 1,300],[WIDTH - 1, 325]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[WIDTH - 1,400],[WIDTH - 1, 425]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[WIDTH - 1,650],[WIDTH - 1, 675]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[900,719],[930, 719]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[500,719],[530, 719]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[220,719],[250, 719]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[0,250],[0,270]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[0,500],[0,520]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[0,700],[0,719]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    pts1 = np.array([[b,a],[750,0],[780, 0]], np.int32)
+    pts1 = pts1.reshape((-1,1,2))
+    cv2.fillPoly(ret,[pts1],(255,255,255))
+
+    return ret
+
+def lens_glare(image, seed):
+    '''
+    Brightens a region to simulate the effect of glare on a camera lens
+
+    INPUTS
+    ------
+    image: Input image of dimensions WIDTH x HEIGHT to be modified.
+    seed: Allows you to repreduce results. If called with the same seed twice, the output will be the same.
+        any integer
+    
+    NOTES
+    -----
+    Does not affect labels.
+    '''
+
+    return apply_mask(image, apply_gaussian(glare_mask(seed), kernel_x = 10, kernel_y = 10), np.full((HEIGHT, WIDTH, 3), 255, dtype=np.uint8))
+
+if __name__ == "__main__":
```

Comment:
General notes
=========

Things that are great!
------------------------
* I like your coding style. Very clean, to the point.
* Great job on making clean, thorough docstrings :smile:

Things to Polish
------------------
**(I admit I'm super picky and sorry)**


* Just general code clean-up would be nice (e.g. use a Linter... I like [Black](https://github.com/psf/black)) -- I should actually implement a general linter for the whole project to ensure consistent styles, I just haven't gotten around to it yet
* Where I expanded the scope of the project, don't worry about getting those done for this MR. Feel free to make a new MR later
* This is a very long file, so maybe break this up into a few separate files (e.g. the initialization, the actual functions you  use, the testing code) -- use your judgment!
* Something that I personally like to use: type hints! It makes the code easier to use correctly. E.g.
```python
def apply_mosaic(image: np.ndarray, seed: int, num_swaps: int) -> np.ndarray:
    ...
```
* More descriptive variable/parameter names!
* Stick to a common convention with variable names (make the code your own, even if you got it from a tutorial/stackoverflow 😛)
* See my other more detailed comments!